pendant/patterns/header.php

```diff
@@ -13,7 +13,7 @@
 
 <!-- wp:site-title {"textAlign":"center","style":{"elements":{"link":{"color":{"text":"var:preset|color|background"}}}}} /-->
 
-<!-- wp:social-links {"iconColor":"background","iconColorValue":"#ffffff","className":"is-style-logos-only"} -->
+<!-- wp:social-links {"iconColor":"background","iconColorValue":"currentColor","className":"is-style-logos-only"} -->
```

Comment:
I think primary would have worked here too, but I don't have a preference, it looks fine in the editor so 👍